libs/ledger-live-common/src/families/cardano/js-synchronisation.ts

```diff
@@ -320,8 +320,11 @@ const makeGetAccountShape =
       }));
     const cardanoNetworkInfo = await getNetworkInfo(initialAccount as CardanoAccount, currency);
     const delegationInfo = await getDelegationInfo(currency, stakeCredential.key);
+
+    let totalBalance = utxosSum;
+
     if (delegationInfo?.rewards) {
-      accountBalance = accountBalance.plus(delegationInfo.rewards);
+      totalBalance = utxosSum.plus(delegationInfo.rewards);
```

Comment:
I would get rid of the `totalBalance` variable (cf. other comment below).

Alternatively, you can keep it but simplify it's definition with:
```ts
const totalBalance = delegationInfo?.rewards ? utxosSum.plus(delegationInfo.rewards) : utxosSum
```

And keep using it below in the return object as:
```ts
return {
  ...
  balance: totalBalance,
  ...
}

```

Reply:
Done